features/step_definitions/redmine_steps.rb

```diff
@@ -73,12 +78,22 @@
 
 When(/^I try to open random project with (\d+) retries$/) do |retries_count|
   @project[:name] = Faker::Hipster.word.capitalize
-  open_random_project(@project[:name], retries_count.to_i)
+  random_project_url = "http://demo.redmine.org/projects/#{@project[:name]}"
+
+  i = 0
```

Comment:
Too much of technical details in step definition 
Pease think about possibility to hide it somewhere
(sometimes it's ok to define some helper methods then wrap them up in module and include it in env.rb for example)